server/test/test-utils.ts

```diff
@@ -0,0 +1,41 @@
+import { getConnection } from 'typeorm';
+import { CanActivate, ExecutionContext } from '@nestjs/common';
+import { TestingModuleBuilder } from '@nestjs/testing';
+import { AuthUserDto } from '../src/decorators/auth-user.decorator';
+import { JwtAuthGuard } from '../src/modules/immich-jwt/guards/jwt-auth.guard';
+
+type CustomAuthCallback = () => AuthUserDto;
+
+export async function initDb() {
+  return getConnection().synchronize(true);
```

Comment:
Does this action create a database in memory or also on the Postgres container?

I've known some problems that the synchronize feature cannot give the same database structure that we are having using migration. Maybe using migration here instead.